front/src/applications/stdcm/views/StdcmView.tsx

```diff
@@ -141,16 +143,17 @@ const StdcmView = () => {
     }
   }, [
     pathProperties,
-    isStdcmResultsEmpty,
+    hasConflicts,
     stdcmResults?.speedSpaceChartData?.formattedPathProperties,
+    stdcmTrainConflicts,
```

Comment:
`stdcmTrainConflicts` is enough

```suggestion
    stdcmResults?.speedSpaceChartData?.formattedPathProperties,
    stdcmTrainConflicts,
```